app.js

```diff
@@ -15,28 +22,32 @@ app.get("/api/articles/:article_id", getArticleById);
 
 app.get("/api/articles", getArticles);
 
-app.get("/api/articles/:article_id/comments", getCommentsForArticle)
+app.get("/api/articles/:article_id/comments", getCommentsForArticle);
+
+app.post("/api/articles/:article_id/comments", postCommentForArticleId);
 
 app.all("*", (req, res, next) => {
-    res.status(404).send({ message: "Endpoint not found"})
-})
+  res.status(404).send({ message: "Endpoint not found" });
+});
 
 app.use((err, req, res, next) => {
-    if (err.status && err.message) {
-        res.status(err.status).send({ message: err.message })
-    }
-    next(err)
-}) 
+  if (err.status && err.message) {
+    res.status(err.status).send({ message: err.message });
+  }
+  next(err);
+});
 
 app.use((err, req, res, next) => {
-    if (err.code === '22P02') {
-        res.status(400).send({ message: "Invalid request"})
-    }
-    next(err)
-})
+  if (err.code === "22P02") {
+    res.status(400).send({ message: "Invalid request: ID has incorrect format" });
+  } else if (err.code === "23502") {
+    res.status(400).send({ message: "Invalid request: Object has incorrect properties" });
+  }
+  next(err);
+});
 
 app.use((err, req, res, next) => {
-    res.status(500).send({ message: "Internal server error"})
-})
+  res.status(500).send({ message: "Internal server error" });
+});
```

Comment:
I think you should move your error handlers to a specific error handler file - it'll stop your app.js from becoming too cluttered :)